level1.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
```

Comment:
Good job on separating the concern of each level into different `html` 👍 

I hope you realized that this is tedious, but this is inevitable in unit 1.